new/LIT7151Miracle.xml

```diff
@@ -0,0 +1,132 @@
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="EMML76">
+                        <desc>see <ref type="mss" corresp="EMML76 ">EMML (HMML) 76, f. 53r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="EMML4205">
+                        <desc>see <ref type="mss" corresp="EMML4205">EMML 4205, ff. 25v, 51r,
+                            26r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LLIT7151Miracle" passive="PULet046  ">
+                        <desc>
+                            <ref type="mss" corresp="PULet046">PULet046, f. 85r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="PULet041">
+                        <desc>
+                            <ref type="mss" corresp="PULet041">PULet041, f. 152r </ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="BLorient639">
+                        <desc>
+                            <ref type="mss" corresp="BLorient639">BLorient639, f.  57v</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="ESdma007">
+                        <desc>
+                            <ref type="mss" corresp="ESdma007">ESdma007, f. 17r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="ESaqm006">
+                        <desc>
+                            <ref type="mss" corresp="ESaqm006">ESaqm006, f. 64r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="IES165">
+                        <desc>
+                            <ref type="mss" corresp="IES165">IES165, f. 43r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive=" EMML642">
+                        <desc>
+                            <ref type="mss" corresp=" EMML642"> EMML642, f. 95r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="Tanasee102">
+                        <desc>
+                            <ref type="mss" corresp="Tanasee102">Tanasee102, f. 70v</ref>
+                        </desc>
+                    </relation>
```

Comment:
you do not need hasAttestation relations but you MUST have relations to show the connections between the different texts